pandas/core/apply.py

```diff
@@ -1794,14 +1794,14 @@ def normalize_keyword_aggregation(
 
 
 def _make_unique_kwarg_list(
-    seq: Sequence[tuple[Any, Any]]
+    seq: Sequence[tuple[Any, Any]],
```

Comment:
this isn't a docstring. did ruff change this?

Reply:
Yeah I believe from ruff's upgrade to `v0.1.13` in this PR